packages/fluentui/react-northstar/src/themes/teams/siteVariables.ts

```diff
@@ -11,7 +11,8 @@ export { colors, contextualColors, naturalColors, primitiveColors, colorScheme,
 // BORDER STYLES
 //
 export const borderWidth = '1px';
-export const borderRadius = '2px';
+export const borderRadiusSmall = '2px';
```

Comment:
Is it worth adding a comment for this one that it's used to keep designs for focus border on inputs?

Reply:
sure